settings/base.py

```diff
@@ -49,7 +61,7 @@ def path(*args):
 STATIC_URL = '/static/'
 
 # Make this unique, and don't share it with anybody.
-SECRET_KEY = '0D8AE44F-5714-40EF-9AC8-4AC6EB556161'
+SECRET_KEY = config('SECRET_KEY')
```

Comment:
Perhaps an `env-dist` file with instructions in the docs to rename to `.env` that includes some defaults? Or I could even wrap the `SECRET_KEY` setting in a try-except and print a more useful error message about creating a .env or setting env vars.